acceptance/api/v1/namespace_auth_test.go

```diff
@@ -0,0 +1,161 @@
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Users Namespace", func() {
+	var request *http.Request
+	var err error
+
+	createNamespace := func(user, password, namespace string) {
+		jsonRequest := fmt.Sprintf(`{"name":"%s"}`, namespace)
+		endpoint := fmt.Sprintf("%s%s/namespaces", serverURL, api.Root)
+
+		request, err = http.NewRequest(http.MethodPost, endpoint, strings.NewReader(jsonRequest))
+		Expect(err).ToNot(HaveOccurred())
+		request.SetBasicAuth(user, password)
+
+		response, err := env.Client().Do(request)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.StatusCode).To(Equal(http.StatusCreated))
+	}
+
+	showNamespace := func(user, password, namespace string) *http.Response {
+		endpoint := fmt.Sprintf("%s%s/namespaces/%s", serverURL, api.Root, namespace)
+		request, err = http.NewRequest(http.MethodGet, endpoint, nil)
+		Expect(err).ToNot(HaveOccurred())
+		request.SetBasicAuth(user, password)
+
+		response, err := env.Client().Do(request)
+		Expect(err).ToNot(HaveOccurred())
+
+		return response
+	}
+
+	Describe("having two user with 'user' role and an admin user", func() {
+		var user1, passwordUser1 string
+		var user2, passwordUser2 string
+		var userAdmin, passwordAdmin string
+
+		BeforeEach(func() {
+			user1, passwordUser1 = env.CreateEpinioUser("user", nil)
+			user2, passwordUser2 = env.CreateEpinioUser("user", nil)
+			userAdmin, passwordAdmin = env.CreateEpinioUser("admin", nil)
+		})
+
+		AfterEach(func() {
+			env.DeleteEpinioUser(user1)
+			env.DeleteEpinioUser(user2)
+			env.DeleteEpinioUser(userAdmin)
+		})
+
+		Describe("each user creates a namespace", func() {
+			var namespaceUser1, namespaceUser2, namespaceAdmin string
+
+			BeforeEach(func() {
+				namespaceUser1 = catalog.NewNamespaceName()
+				createNamespace(user1, passwordUser1, namespaceUser1)
+
+				namespaceUser2 = catalog.NewNamespaceName()
+				createNamespace(user2, passwordUser2, namespaceUser2)
+
+				namespaceAdmin = catalog.NewNamespaceName()
+				createNamespace(userAdmin, passwordAdmin, namespaceAdmin)
+			})
+
+			AfterEach(func() {
+				env.DeleteNamespace(namespaceUser1)
+				env.DeleteNamespace(namespaceUser2)
+				env.DeleteNamespace(namespaceAdmin)
+			})
+
+			When("user1 try to show a namespace", func() {
+				It("can show his namespace", func() {
+					response := showNamespace(user1, passwordUser1, namespaceUser1)
+					Expect(response.StatusCode).To(Equal(http.StatusOK))
+					response.Body.Close()
+				})
+
+				It("cannot show user2 namespace", func() {
+					response := showNamespace(user1, passwordUser1, namespaceUser2)
+					Expect(response.StatusCode).To(Equal(http.StatusUnauthorized))
+					response.Body.Close()
+				})
+
+				It("cannot show userAdmin namespace", func() {
+					response := showNamespace(user1, passwordUser1, namespaceAdmin)
+					Expect(response.StatusCode).To(Equal(http.StatusUnauthorized))
+					response.Body.Close()
+				})
+			})
+
+			When("user2 try to show a namespace", func() {
```

Comment:
I think `user1` and `user2` are symmetric in this test, we don't need to test them both. There is nothing special about user1 compared to user2.